packages/nimble-components/src/table/models/table-validator.ts

```diff
@@ -105,4 +107,18 @@ export class TableValidator<TData extends TableRecord> {
 
         return !this.duplicateSortIndex;
     }
+
+    public validateColumnGroupIndices(groupIndices: number[]): boolean {
+        this.duplicateGroupIndex = false;
+
+        const groupIndexSet = new Set<number>();
```

Comment:
Since this implementation is identical to `validateColumnSortIndices`, maybe there should be a private helper function in this class that validates that an array of numbers all have unique values.